From 31ce6b9926431ece69d97f4593db4fe53be87eae Mon Sep 17 00:00:00 2001
From: Jinzhe Zeng <jinzhe.zeng@rutgers.edu>
Date: Fri, 6 Aug 2021 03:27:01 -0400
Subject: [PATCH] remove unused reference

---
 AmberTools/src/sander/ml.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/AmberTools/src/sander/ml.cpp b/AmberTools/src/sander/ml.cpp
index 223d3c8359..bc887c8494 100644
--- a/AmberTools/src/sander/ml.cpp
+++ b/AmberTools/src/sander/ml.cpp
@@ -17,7 +17,6 @@
 #endif
 
 #ifdef MLCUDA
-#include <builtin_types.h>
 // # i nclude <drvapi_error_string.h>
 #endif
 
@@ -42,7 +41,6 @@ namespace ml
 
 #define MAGIC_NUMBER 256
 typedef double compute_t;
-#include <tensorflow/core/public/session.h>
 
 //static
 //void
-- 
2.31.1

